Client/sdk/game/CClock.h

```diff
@@ -16,4 +16,8 @@ class CClock
 public:
     virtual void Set(BYTE bHour, BYTE bMinute) = 0;
     virtual void Get(BYTE* bHour, BYTE* bMinute) = 0;
+
+    virtual bool SetTimerCycle(bool value) noexcept = 0;
```

Comment:
Please rename functions also in the CClock class

Reply:
fixed in (https://github.com/multitheftauto/mtasa-blue/pull/3567/commits/8213d4868071c5a2e0578c854cd28008048dc0ab)